hudi-cli/src/main/java/org/apache/hudi/cli/commands/ArchivedCommitsCommand.java

```diff
@@ -206,4 +250,22 @@ private Comparable[] readCommit(GenericRecord record, boolean skipMetadata) {
       return new Comparable[] {};
     }
   }
+
+  private void initJavaSparkContext() {
```

Comment:
we should have existing util methods to instantiate right. can we try re-using it? how other commands are initializing jsc?

Reply:
I pretty much copied from MetadataCommand.java, but got rid of userDefinedMaster arg:
```
private void initJavaSparkContext(Option<String> userDefinedMaster) {
    if (jsc == null) {
      jsc = SparkUtil.initJavaSparkContext(SparkUtil.getDefaultConf("HoodieCLI", userDefinedMaster));
    }
  }
```

Reply:
not sure I understand. I only see 
```
  private void initJavaSparkContext(Option<String> userDefinedMaster) {
    if (jsc == null) {
      jsc = SparkUtil.initJavaSparkContext(SparkUtil.getDefaultConf("HoodieCLI", userDefinedMaster));
    }
  }
```
in MetadataCommand